src/content/docs/reference/stages.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Stages
+description: A stage is a separate image build flow executed in parallel with the main build.
+---
+
+:::note
+This feature is currently only available when using `use_unstable_cli` option in the [GHA](/reference/github-action/)
+:::
```

Comment:
IMO the docs should only be merged after a version of the CLI and action with these features is released.

Reply:
That's a fair point

Reply:
However, I would also argue that it could be in here. Docker's docs have information about their experimental features. They just always make sure to note that it's an experimental feature. If we merge this, then the docs would be available to users who would like to try new features and then we could possibly get some feedback.

It's up to you whether this gets merged in though.